### PR TITLE
React react2 week1

### PR DESCRIPTION
**Pull request by Kositthai**
Submit Meal-Sharing react-react2-week1

### CODE REVIEW

File: src/client/components/Meal.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+import mealCard from "./styles/mealCard.css";
+import foodPics from "../../../public/foodPics.jpg";
+
+export default function Meal() {
+  
+  const { meals } = useContext(MealContext);
+
+  return (
+      <div className="meal-container">
+        {meals?.map((meal) => {
```
Comment (shpomp) [inline src/client/components/Meal.js:12]: nice 🤓

File: src/client/components/Meal.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
+import { MealContext } from "./MealContext";
+import React, { useContext } from "react";
+import mealCard from "./styles/mealCard.css";
+import foodPics from "../../../public/foodPics.jpg";
+
+export default function Meal() {
+  
+  const { meals } = useContext(MealContext);
```
Comment (shpomp) [inline src/client/components/Meal.js:8]: Try destructuring the properties 💡

**Review by shpomp — APPROVED** [review]
Very nice, and with some extra things too! 
Great job 👏🏽 🌻

**Comment by Kositthai** [comment]
> Very nice, and with some extra things too! Great job 👏🏽 🌻

Thank you so much for your feedback and your time :)

**Pull request merged**